### PR TITLE
Correct typo in LockPool and make ResourcePool max size configurable

### PR DESCRIPTION
**Pull request by jiacheliu3**

### CODE REVIEW

File: core/common/src/main/java/alluxio/collections/LockPool.java [hunk 1]
```diff
@@ -59,17 +59,7 @@
   private static final String EVICTOR_THREAD_NAME = "LockPool Evictor";
 
   private final Map<K, Resource> mPool;
-  // TODO(baoloongmao): make it configurable before merge this PR.
-  private final ResourcePool<Resource> mResueLockPool = new ResourcePool(1000000) {
```
Comment (jiacheliu3) [inline core/common/src/main/java/alluxio/collections/LockPool.java:63]: Not sure if this intends to be `reused` or `resource`

Comment (maobaolong) [reply core/common/src/main/java/alluxio/collections/LockPool.java:63]: Sorry, It should be `reuse` as you have guessed what I mean 😄

**Review by maobaolong — APPROVED** [review]
@jiacheliu3 Thank you very much to help me correct my typo issue, LGTM.

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**